lib/rubocop/cop/style/symbol_array.rb

```diff
@@ -69,9 +69,9 @@ def on_array(node)
 
         def complex_content?(node)
           node.children.any? do |sym|
-            content, = *sym
-            content = content.to_s
-            content_without_delimiter_pairs = content.gsub(/(\[\])|(\(\))/, '')
+            content = *sym
+            content = content.map { |c| c.respond_to?(:str_content) ? c.str_content : c.to_s }.join
```

Comment:
Can you refactor to below?

```suggestion
            content = content.map { |c| c.is_a?(AST::Node) ? c.source : c }.join
```

Reply:
Yes. That is much clearer. I was looking for a more concise way to check this other than using `respond_to?`. Done